asf_search/WKT/RepairEntry.py

```diff
@@ -0,0 +1,7 @@
+class RepairEntry:
+    def __init__(self, report_type: str, report: str) -> None:
+        self.report_type = report_type
+        self.report = report
+    
+    def __str__(self) -> str:
+        return f'{self.report_type}\{self.report}'
```

Comment:
My linter says this is invalid, unless I escape with a second back slash. Is that right?